### DISCUSSION TITLE
[活动] 5月 TODO APAC Sync Call

### DISCUSSION THREAD

**Discussion by xiaoya-yaya** [kaiyuanshe/ONES#103] [TODO & OSPO Alliance chapter]
Time：5月30 - 16:00 – 17:00，周一
Zoom：https://zoom.us/j/93975546839?pwd=eGpldW05VDFkMkNKWkMxWGFaUHVDQT09
ID: 939 7554 6839
Passcode: 140388

### About TODO APAC Sync call 
 
TODO APAC Sync meetings were created with the aim of working together to improve OSPO adoption and education within the different APAC regions and discuss with the broader community the challenges these organizations face when implementing an open-source program or a similar initiative, following the Chatham House Rules.

**Comment by xiaoya-yaya** [comment]
会议记录：https://github.com/todogroup/ospology/blob/main/meetings/Sync%20Chapters/APAC-notes/May-Chinese.md

## 话题

* OSPO 思维导图 v2.0 review
* OSPO 问卷
* 和欧洲或美国相比，在亚洲建立 OSPO 有哪些特别的挑战？ 
* 将使用/编写开源软件作为商业工具：如何说服亚洲区域的 c-level 管理。

## 会议纪录

* OSPO 思维导图: https://ospomindmap.todogroup.org/
   * 大家可以在 PR 上 review
* OSPO 问卷
   * 现在还可以在问卷上增加问题吗？
      * 离发布时间很接近了（6月22），但应该还可以做到
   * 关于问卷流程的问题：如何在 APAC 地区传播？
      * 问卷会被翻译为多语言，通过 Survey Monkey 生成专有链接
      * Ana 会再问问 Hilary 有关流程的变更
* 和欧洲或美国相比，在亚洲建立 OSPO 有哪些特别的挑战？ 
    * 和欧洲一样，很多事情都在很早期阶段。有些地方不一定把这个开源相关的团队称作 OSPO，但是都在做围绕许可证，安全，合规的事情。缺乏围绕社区的事情。
    * 如：在印度也需要合规培训，问题是类似的。
    * 文化差异是一个巨大挑战，很难让亚洲的参与者迈出第一步。
    * C-level 管理者常常不理解贡献上游的价值，这会成为一个组织继续开源的巨大阻碍。
    * 简单的题到了这个工具（ORT）: https://github.com/oss-review-toolkit/ort 
    * 缺乏开源文化（公司抵触使用“免费”的东西）
* 课程能否更好地继承进企业的内部开源培训流程，来减轻开源文化教育和 OS 101？有没有亚洲公司所采取的有价值的措施？ 
    * 培训课程可以面向具体的国家，并且要了解每个国家的不同背景。
* 如何说服亚洲区域的 c-level 引入开源
    * 将使用/编写开源软件作为商业工具 (面向 c-level 执行)
    * 不同区域和中层管理者的交互方式是不同的
    * 对本土开发者来说，在组织内扩展非常重要。
    * 关键是了解每个地域的语境，比如说，在西方 vs 亚洲，如何培训。
    * 了解当地语言的本土大使非常重要，我们需要这种连接。

**Reply by will-ww** [reply]
太棒，很多事情可以对接起来，我们一起来讨论~
